packages/styles/scss/patterns/simplelongform/README.md

```diff
@@ -0,0 +1,35 @@
+### SCSS
+
+#### Usage
+
+Import the package css into the top of your main CSS file.
+
+```css
+@import '@carbon/ibmdotcom-styles/scss/patterns/leadspace/index';
+```
+
+#### Modifiers
+
+Use these modifiers with `.bx--leadspace` class.
+
+| Class                        | Description                                                        |
+| ---------------------------- | ------------------------------------------------------------------ |
+| `.bx--leadspace--productive` | The className for 'productive' variant which changes title styling |
+
+#### Elements
+
+| Class                       | Description                                            |
+| --------------------------- | ------------------------------------------------------ |
+| `.bx--leadspace__image`     | Background image of the lead space pattern (optional)  |
+| `.bx--leadspace__container` | Element containing all copy/cta content                |
+| `.bx--leadspace__overlay`   | Gradient overlay behind copy content                   |
+| `.bx--leadspace__title`     | Required title/heading for the lead space              |
+| `.bx--leadspace__content`   | Content holder for optional short description and ctas |
+| `.bx--leadspace__desc`      | Optional short description                             |
+| `.bx--leadspace__ctas`      | Optional CTA buttons                                   |
```

Comment:
README is for leadspace?

Reply:
my bad, copy & paste.
updating it

Reply:
code updated with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/717dd6b17118b19ee6179246a74f58cb34783d70)

Reply:
updated the code with [this commit](https://github.com/carbon-design-system/ibm-dotcom-library/pull/561/commits/5cf530ae093831932f0b041ef4b6ae2326cfd812)